RF24Mesh_config.h

```diff
@@ -64,6 +64,23 @@
     #define MESH_CONNECTION_CHECK_ATTEMPTS 3
 #endif
 
+/**
+ * @brief How to verify a connection
+ *
+ * On child nodes, determine how they verify/check their connection periodically, or when writes start to fail via the `mesh.checkConnection();`
+ * function.
+ * Set RF24MESH_CONN_CHECK_TYPE to RF24MESH_CONN_CHECK_PARENT for the new behaviour of verifying connectivity only with their parent node
+ * Set RF24MESH_CONN_CHECK_TYPE to RF24MESH_CONN_CHECK_MASTER for the old behaviour of verifying connectivity with the master node
+ * The old behaviour typically results in more network congestion, more load on the master node, and less reliable networks,
+ * but can work well when radio conditions are good and/or when there are only a small number of nodes on the network and/or in close proximity
```

Comment:
```suggestion
 * but it can work well when radio conditions are good and/or when there are only a small number of nodes on the network and/or in close proximity
```
## General rule of conjunctions
If a conjunction is followed by a subject-verb agreement, then the conjunction is preceded by a comma.
Both examples below are correct/proper: 
```
This is good but can be grammatically confusing. 

This is good, but it can be separated into simple sentences.
```

The fact that the conjunction is following a comma-separated list adds complexity. Without a subject-verb agreement after the conjunction, there would be no need for a comma after the list, and the conjoined idea could be mistakenly read as part of the last element of the list.

Reply:
resolved in https://github.com/nRF24/RF24Mesh/pull/250/commits/4b1e3a2895f35db40284158248669d22a98d2781